src/App.js

```diff
@@ -134,13 +134,17 @@ class App extends Component {
 
                         {
                           athlete.scores.map((score) => {
-                            return (
-                              <td>
-                                <strong>{ this.ordinalSuffix(score.rank) }</strong>
-                                <br />
-                                <small>({ score.scoreDisplay })</small>
-                              </td>
-                            )
+                            if (score.rank === 0) {
```

Comment:
```js
score.rank ?
  return (<td />)
  : return (
    <td>
      <strong>{ this.ordinalSuffix(score.rank) }</strong>
      <br />
      <small>({ score.scoreDisplay })</small>
    </td>
  )
```

Reply:
When I added the ternary operator a Syntax error (unexpected token) is raised on 'return' token.

Reply:
Fixed! Please check the last commit